draft-ietf-rtcweb-jsep.xml

```diff
@@ -2062,11 +2063,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction MUST be "send". If the application has chosen a
               RID value, it MUST be used as the rid-identifier;
               otherwise a RID value MUST be generated by the
-              implementation. When generating RID values, it is
-              RECOMMENDED that the values be 3 bytes or less, to allow
-              them to efficiently fit into the RTP header extension
-              defined in
-              <xref target="I-D.ietf-avtext-rid"></xref>, Section 11.
+              implementation. RID values MUST be generated in a fashion that does not leak user
+              information, e.g., randomly or using a per-PeerConnection counter, and
+              SHOULD be 3 bytes or less, to allow them to efficiently fit into
+              the RTP header extension defined in
+              <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation">
```

Comment:
I think you need I-D.ietf-avtext-rid here; maybe the xref above needs its section changed as well.